app/src/js/components/Collections/collection-header.js

```diff
@@ -25,7 +25,7 @@ const CollectionHeader = ({
     },
     {
       label: 'Collections',
-      href: '/collections',
+      href: '/collections/all',
```

Comment:
Changed to `/collections/all` because `/collections` redirects to `/collections/all` and all query params were stored for `/collections/all`, but for <Link>s to `/collections` params were not appended.